src/flatc.cpp

```diff
@@ -121,6 +121,11 @@ std::string FlatCompiler::GetUsageString(const char *program_name) const {
     "                         (see the --cpp-str-flex-ctor option to change this behavior).\n"
     "  --cpp-str-flex-ctor    Don't construct custom string types by passing std::string\n"
     "                         from Flatbuffers, but (char* + length).\n"
+    "  --cpp-field-case STYLE Generate C++ fields using selected case style.\n"
+    "                         Supported STYLE values:\n"
+    "                          * 'unchanged' - leave unchanged (default);\n"
+    "                          * 'upper' - schema snake_case emits UpperCamel;\n"
+    "                          * 'lower' - schema snake_case emits lowerCamel.\n"
```

Comment:
The `--cpp-field-case`  looks like `--cpp-field-style` instead of `-case`.
- 'unchanged' - leave unchanged (default);
- 'upper_camel' -upper camel case;
- 'lower_camel' - lower camel case.

Reply:
will rename to cpp-field-case-style